examples/landmark_detection/example_scan.py

```diff
@@ -0,0 +1,218 @@
+from giskard_vision.landmark_detection.marks.facial_parts import FacialParts
+from giskard_vision.scanner.scanner_vision import Scanner
+
+from giskard.scanner.issues import Issue, IssueLevel, IssueGroup
+
+# %%
+
+
+class FacialPartsDetector:
+    def __init__(self):
+        pass
+
+    def run(self, model, dataset):
+        facial_parts = [FacialParts.LEFT_HALF.value, FacialParts.RIGHT_HALF.value]
+
+        issues = []
+
+        for fp in facial_parts:
+            dl = CroppedDataLoader(dataset, part=fp)
+            result = TestDiff(metric=NMEMean, threshold=1).run(
+                model=model,
+                dataloader=dl,
+                dataloader_ref=dataset,
+                facial_part=fp,
+            )
+            result = result.to_dict()
+
+            level = IssueLevel.MAJOR if result["metric_value"] > 0 else IssueLevel.MEDIUM
+            issues.append(
+                Issue(
+                    model,
+                    dataset,
+                    level=level,
+                    slicing_fn=f"Facial part == {fp.name}",
+                    group=IssueGroup("Robustness to cropping", "Warning"),
+                    meta={
+                        "metric": result["metric"],
+                        "metric_value": result["metric_value"],
+                    },
+                )
+            )
+
+        return issues
+
+
+class ResizedImagesDetector:
+    def __init__(self, transformation="scaling", scales=0.5, kernel_size=(11, 11), sigma=(3, 3)):
+        self.transformation = transformation
+        self.scales = scales
+        self.kernel_size = kernel_size
+        self.sigma = sigma
+
+    def run(self, model, dataset):
+        if self.transformation == "scaling":
+            dl = ResizedDataLoader(dataset, scales=self.scales)
+            slicing_fn = f"scaling = {self.scales}"
+        elif self.transformation == "color":
+            dl = ColoredDataLoader(dataset)
+            slicing_fn = f"color = grayscale"
+        elif self.transformation == "blurr":
+            dl = BlurredDataLoader(dataset, self.kernel_size, self.sigma)
+            slicing_fn = f"blurr = kernel {self.kernel_size}, sigma {self.sigma}"
+
+        result = TestDiff(metric=NMEMean, threshold=1).run(
+            model=model,
+            dataloader=dl,
+            dataloader_ref=dataset,
+        )
+        result = result.to_dict()
+        level = IssueLevel.MAJOR if result["metric_value"] > 0 else IssueLevel.MEDIUM
+        issues = []
+
+        issues.append(
+            Issue(
+                model,
+                dataset,
+                level=level,
+                slicing_fn=slicing_fn,
+                group=IssueGroup("Robustness to transformations", "Warning"),
+                meta={
+                    "metric": result["metric"],
+                    "metric_value": result["metric_value"],
+                },
+            )
+        )
+        return issues
+
+
+class HeadPoseDetector:
```

Comment:
should inherit from: https://github.com/Giskard-AI/giskard/blob/1882fb6fa617216266d9a8b1fd7fd6307f43a365/giskard/scanner/registry.py#L11-L14